### PR TITLE
Message notification anchor

### PR DESCRIPTION
**Pull request by MigDinny**
Now:
image

Before:
image

This is a continuation of the messages string pull request #784 suggested by @teryhill 

After many hours of working and understand how knockout works, there it is.

Issue: #779

### CODE REVIEW

**Comment by robbyoconnor** [comment]
Remember to approve the task when you agree the work is completed to your satisfaction :)

File: interface/main/tabs/js/user_data_view_model.js [hunk 1]
```diff
@@ -36,4 +36,11 @@ function userPrefs()
...
     navigateTab(webroot_url+"/interface/super/edit_globals.php?mode=user","msc");
     activateTabByName("msc",true);
+}
+
+// this function is calling when clicking on "x unread messages" anchor on upper right corner
+// this opens the messages tab
+function unreadMessages() {
+    navigateTab(webroot_url+"/interface/main/messages/messages.php?form_active=1", "pat");
+    activateTabByName("pat", true);
 }
```
Comment (aethelwulffe) [inline interface/main/tabs/js/user_data_view_model.js]: Please add a new line to normalize this file.  It helps when we clean up this stuff as we find it.

File: interface/main/tabs/templates/user_data_template.php [hunk 2]
```diff
@@ -14,10 +26,11 @@
                     <?php } ?>
                     <li data-bind="click: userPrefs"><?php echo xlt("User Preferences");?></li>
                     <li data-bind="click: changePassword"><?php echo xlt("Change Pass Phrase");?></li>
```
Comment (aethelwulffe) [inline interface/main/tabs/templates/user_data_template.php]: Another thing to look for is php tags crammed up against other stuff.  It is fine here, but there are cases where it breaks stuff (like in query builders).

**Comment by aethelwulffe** [comment]
It looks like @Trodrige has his approval on this as well, making three reviews.  I just tested with a hyphenated name as some other silly stuff, and it still shows perfectly.  I like.

**Pull request merged**